victor/src/main/groovy/com/trello/victor/SVG2AndroidTask.groovy

```diff
@@ -0,0 +1,76 @@
+package com.trello.victor
+
+import com.romainpiel.svgtoandroid.Svg2Vector
+import org.gradle.api.DefaultTask
+import org.gradle.api.GradleException
+import org.gradle.api.file.FileCollection
+import org.gradle.api.tasks.InputFiles
+import org.gradle.api.tasks.OutputDirectory
+import org.gradle.api.tasks.TaskAction
+import org.gradle.api.tasks.incremental.IncrementalTaskInputs
+import org.gradle.api.tasks.incremental.InputFileDetails
+
+class SVG2AndroidTask extends DefaultTask {
+
+    /**
+     * The input SVGs.
+     */
+    @InputFiles
+    FileCollection sources
+
+    /**
+     * The output directory.
+     */
+    @OutputDirectory
+    File outputDir
+
+    @TaskAction
+    def transform(IncrementalTaskInputs inputs) {
```

Comment:
So much of this code is the exact same as the other one - the only part that's different is the actual conversion.

I think this is probably fine for an initial pull, but do you think it'd work to just merge these into a single task that has a generator method provided to it?